libs/safe/src/lib/components/widgets/editor/editor.component.ts

```diff
@@ -59,6 +59,7 @@ export class SafeEditorComponent implements OnInit {
   /** Sanitizes the text. */
   ngOnInit(): void {
     this.setContentFromLayout();
+    this.formattedStyle = this.settings.widgetDisplay.style;
```

Comment:
```suggestion
    this.formattedStyle = this.settings.widgetDisplay?.style;
```
For empty text widgets:
![image](https://github.com/ReliefApplications/oort-frontend/assets/123092672/48fb8611-aed6-4813-a377-b617cbd1ad6c)